core/src/test/java/org/elasticsearch/action/IndicesRequestIT.java

```diff
@@ -66,6 +66,7 @@
 import org.elasticsearch.action.index.IndexRequest;
 import org.elasticsearch.action.search.SearchRequest;
 import org.elasticsearch.action.search.SearchResponse;
+import org.elasticsearch.action.search.SearchTransportService;
```

Comment:
Hm... you want me revert and leave the imports not organized?

In general I agree that major reformatting should happen separately but it is just a single import ;)